src/views/open-api-docs-view/types.ts

```diff
@@ -32,9 +20,14 @@ export interface OperationProps {
 		full: string
 		truncated: string
 	}
-	queryParameters?: PropertyDetailProps[]
-	pathParameters?: PropertyDetailProps[]
-	bodyParameters?: PropertyDetailProps[]
+	requestData: {
+		heading: string
+		propertyDetails: PropertyDetailsProps[]
+	}[]
+	responseData: {
+		heading: string
+		propertyDetails: PropertyDetailsProps[]
+	}[]
```

Comment:
Do you want to unify any types here between the interface used with the `OperationDetails` component?

Reply:
Ah yeah, great call! 🙌 Have updated to do that 🙇‍♂️